autopush/router/gcm.py

```diff
@@ -12,7 +12,7 @@
 class GCMRouter(object):
     """GCM Router Implementation"""
     log = Logger()
-    gcm = None
+    gcm = {}
```

Comment:
We only instantiate one GCMRouter per app it looks like, but let's avoid this evil mutable class variable regardless